packages/react-native-editor/package.json

```diff
@@ -108,7 +108,7 @@
 		"device-tests-canary": "cross-env NODE_ENV=test jest --forceExit --detectOpenHandles --no-cache --maxWorkers=2 --testPathPattern=@canary --config ./jest_ui.config.js",
 		"device-tests:local": "cross-env NODE_ENV=test jest --runInBand --detectOpenHandles --forceExit --config ./jest_ui.config.js",
 		"device-tests:debug": "cross-env NODE_ENV=test node $NODE_DEBUG_OPTION --inspect-brk node_modules/jest/bin/jest --runInBand --detectOpenHandles --verbose --config ./jest_ui.config.js",
-		"appium:setup": "output=`appium driver list --installed --json` && (echo $output | grep -q 'uiautomator2' && echo 'uiautomator2 is installed, skipping installation.' || (echo 'uiautomator2 not found, installing...' && appium driver install uiautomator2)) && (echo $output | grep -q 'xcuitest' && echo 'xcuitest is installed, skipping installation.' || (echo 'xcuitest not found, installing...' && appium driver install xcuitest))",
+		"appium:setup": "./bin/appium-setup-drivers.sh",
```

Comment:
What do you think about naming these developer-facing scripts to focus on the high-level subject rather than the tooling or implementation details? 

```suggestion
		"test:e2e:setup": "./bin/appium-setup-drivers.sh",
```

The same might be applied to the script file name itself — i.e. `appium-setup-drivers.sh` → `test-e2e-setup.sh` — but I'm less concerned about that.

Reply:
I like this idea! Thanks for the suggestion, I will update both 👍

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/55257/commits/d87bbf05a747e3f6618bebb202bfedd17a04b9ec